member_database/events/__init__.py

```diff
@@ -222,6 +223,43 @@ def resend_email():
     return redirect(url_for('events.index'))
 
 
+@events.route('/resend_emails/', methods=['GET', 'POST'])
+def resend_emails():
+    '''Resend all emails for open events for a given email address'''
+
+    class ResendForm(FlaskForm):
+        email = EmailField(validators=[DataRequired()])
+        submit = SubmitField('Emails für aktuelle Anmeldungen erneut versenden.')
```

Comment:
Do we have any rules where to put custom forms? It might be confusing to have this form in-line vs the `PersonEditForm` [here](https://github.com/pep-dortmund/member-database/blob/master/member_database/forms.py#L8).

Reply:
Not really no. This is a small three liner. The edit form might become large. `events/forms.py` would probably best as soon as we have more than one.